lib/evss/intent_to_file/intent_to_file.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+module EVSS
+  module IntentToFile
+    class IntentToFile < Common::Base
+      # The spelling of these status types has been validated with the partner team
```

Comment:
This seems to be a trend, PPIU also has spelling errors 😞- they should really fix it, and I'll check if they can (their excuse is that their sub-system also has spelling errors). If not we may have to transform it back and forth.